scalableminds/webknossos
# Add possibility to show random tasks

### Mailable description of changes:
 - The admin has now the possibility to show the task in a random order. This feature is used when you click on 'Show random subset'. The behavior of 'Search' is still the same.
### Steps to test:
- go to Administration >> Tasks
   - e.g. select Project
   - click 'Search': the result should stay the same every time you click on it
   - click 'Show random subset': the order of the result should change every time you click it (if you have at lest 2 Tasks that fits the criteria)

### Issues:
- fixes #2803 

------
